src/pages/settings/Report/ReportSettingsPage.js

```diff
@@ -148,7 +151,7 @@ class ReportSettingsPage extends Component {
                         )}
                         {shouldAllowWriteCapabilityEditing ? (
                             <MenuItemWithTopDescription
-                                shouldShowRightIcon
+                                shouldShowRightIcon={!ReportUtils.isAdminRoom(this.props.report)}
```

Comment:
@dukenv0307 Why is this change necessary? I don't think this menuitem will be displayed when `ReportUtils.isAdminRoom(this.props.report)` is true.